doc/api/child_process.md

```diff
@@ -831,7 +831,11 @@ pipes between the parent and child. The value is one of the following:
    file descriptor is duplicated in the child process to the fd that
    corresponds to the index in the `stdio` array. The stream must have an
    underlying descriptor (file streams do not until the `'open'` event has
-   occurred).
+   occurred). Because the stream is already piped (data is piped from the
+   stdio to the stream), thus stopping its readable side, otherwise we
+   might read data from the stream when at the same time the child
+   process does. If you want to pass a handle to child process, you can
```

Comment:
I'm not sure what the sentence starting with "Because" is saying. Is anyone else having the same problem? Does it need to be re-worded a little bit?

Reply:
thank you very much!I'll check around

Reply:
"Because the stream is already piped (data is piped from the stdio to the stream)" replace by "Because the stream is already piped to stdio". Would this be better?

Reply:
I think that's a bit more clear but it's still not a complete sentence and mildly puzzling. Maybe it's the "Because" that's throwing me off? Does this seem correct to you?

```suggestion
   occurred). The stream is already piped to stdio, thus stopping its readable
   side. Otherwise we might read data from the stream at the same time that the
   child process does. If you want to pass a handle to child process, you can
```

Reply:
Thanks for you help! You consideration is correct, I didn't describe it clearly. Or how about this?
"we need to stop its readable side, because the stream is already piped to the stdio."

Reply:
I fell like there's a subtly I might be missing here. Maybe someone from @nodejs/child_process can take a look and confirm or suggest language?

Reply:
> I fell like there's a subtly I might be missing here. Maybe someone from @nodejs/child_process can take a look and confirm or suggest language?

Maybe someone from @nodejs/documentation?